src/support/components/FreeAccountSupportOverlay.tsx

```diff
@@ -24,7 +25,42 @@ const FreeAccountSupportOverlay: FC<OwnProps> = () => {
         onDismiss={onClose}
       />
 
-      <Overlay.Body></Overlay.Body>
+      <p className="status-page-text">
+        <span>
+          {' '}
+          <Icon glyph={IconFont.Info_New} />{' '}
+        </span>
+        Check our{' '}
+        <SafeBlankLink href="https://status.influxdata.com">
+          status page
+        </SafeBlankLink>{' '}
+        to see if there is an outage impacting your region.
+      </p>
+      <Overlay.Body>
+        <p>
+          Contact support is currently only available for Usage-Based Plan and
+          Annual customers. Please try our community resources below to recieve
+          help or file a feedback & questions form.
+        </p>
+        <List className="support-links">
+          <List.Item>
+            <div className="community-logo" />
+            <SafeBlankLink href="https://community.influxdata.com">
+              InfluxDB Community Forums
+            </SafeBlankLink>
+          </List.Item>
+          <List.Item>
+            <div className="slack-logo" />
+            <SafeBlankLink href="https://influxcommunity.slack.com/join/shared_invite/zt-156zm7ult-LcIW2T4TwLYeS8rZbCP1mw#/shared-invite/email">
```

Comment:
i also found this link used [here](https://github.com/influxdata/ui/blob/master/src/pageLayout/containers/TreeNav.tsx) - maybe a good time to put it into a shared constants file